packages/shell-api/src/integration.spec.ts

```diff
@@ -492,9 +495,13 @@ describe('Shell API (integration)', function() {
     });
 
     describe('stats', () => {
+      const extraStatsKeys: string[] = [];
```

Comment:
I think this is not really necessary. I believe i've done this test initially, and i was probably a bit lazy and only copy/pasted the result that matched.

Honestly we can just use a different strategy and match only few keys, just to make sure the result is a 'stats' result and not some other command, after all we don't fiddle with the command result in any way.

Long story short, i think is easier and more stable to just remove  `extraStatsKeys` and avoid having to support the differences between versions here.

Reply:
Sure, I mean, that’s also easier for me :slightly_smiling_face: